README.md

```diff
@@ -52,6 +52,47 @@ npm install @google-cloud/recaptcha-enterprise
 ```
 
 
+### Using the client library
+
+```javascript
+/**
+ * Instantiate the reCAPTCHA Enterprise client and make request.
+ * @param {string} projectNumber The project number to use when making requests.
+ */
+async function quickstart(projectNumber) {
```

Comment:
You want an outer function called `main`, and an inner function called `quickstart`:
https://github.com/googleapis/nodejs-asset/blob/master/samples/quickstart.js